src/components/checks/create/(sections)/OverviewSection.tsx

```
@@ -8,13 +8,15 @@ export function OverviewSection() {
-        <p className='dark:text-neutral-300'>Here is a brief overview of what changes were made</p>
+        <p className='text-neutral-600 dark:text-neutral-300'>Here is a brief overview of what changes were made</p>
       </div>
-      <div className='3xl:grid-cols-3 grid gap-8 xl:grid-cols-2 **:dark:[&_input]:ring-neutral-600/70 **:dark:[&_textarea]:ring-neutral-600/70' inert>
+      <div
+        className='3xl:grid-cols-3 grid gap-8 xl:grid-cols-2 **:[&_input]:ring-neutral-300/70 **:dark:[&_input]:ring-neutral-600/70 **:[&_textarea]:ring-neutral-300/70 **:dark:[&_textarea]:ring-neutral-600/70'
+        inert>
         <GeneralSection />
         <QuestionsSection />
         <SettingsSection />
-        <Card disableHoverStyles children className='3xl:hidden' />
+        <Card disableInteractions children className='3xl:hidden' />
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)